lib/features/login/presentation/widgets/long_button.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+import '../../../../utilities/constants.dart';
+
+typedef _LongButtonAction();
```

Comment:
What is the purpose of this?

Reply:
I'll put the definition in the method signature since its an extremely simple function. But the idea is to give type safety to closures inside method calls since Dart allows us to write simply `Function` and accept all types of functions as arguments.

Change on [4b3a8a8](https://github.com/rishFilet/swipe_n_swap/pull/2/commits/4b3a8a80c2eb851c0ef3590e41261c3b56722a5f)